pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -1159,6 +1159,14 @@ func (m *pgDumpReader) readFile(
 					conv.TargetColOrds.Add(idx)
 					targetColMapIdx[j] = idx
 				}
+				// For any missing columns, fill those to NULL.
```

Comment:
We'll want this on master too if we enable IMPORT INTO ... PGDUMP, right?

Reply:
Yep good point, will forward port and also check if it needs to go to 21.1.